pkg/worker/base_runc_config.json

```diff
@@ -24,49 +24,65 @@
 				"CAP_AUDIT_WRITE",
 				"CAP_KILL",
 				"CAP_NET_RAW",
+				"CAP_NET_BIND_SERVICE",
+				"CAP_SYS_ADMIN",
 				"CAP_CHOWN",
 				"CAP_DAC_OVERRIDE",
 				"CAP_FSETID",
 				"CAP_FOWNER",
 				"CAP_SETGID",
 				"CAP_SETUID",
-				"CAP_SETFCAP"
+				"CAP_SETFCAP",
+				"CAP_MKNOD",
+				"CAP_SYS_CHROOT"
```

Comment:
I think the only one of these I actually need is `CAP_SYS_CHROOT`. Will test and revert the other changes if so.